src/st-flash/flash.c

```diff
@@ -126,21 +136,20 @@ int main(int ac, char** av) {
         goto on_error;
     }
 
-    if (o.cmd == FLASH_CMD_WRITE) {                                             // write
+    if (FLASH_CMD_WRITE == o.cmd)                                               // write
+    {
```

Comment:
Style (minor): Can we move the bracket { behind the previous argument?

Reply:
fixed